### PR TITLE
Create presses-universitaires-de-rennes-collection-archeologie-et-cul…

### PR DESCRIPTION
**Pull request by FrancoisC37**
…ture.csl
Hello,
the Presses Universitaires de Rennes have two différents rules for authors (http://pur-editions.fr/consignes.php). With two bibliographic style : the second is specific for archeologist.

The first style already exist in the Zotero Style Repository.
I've done the second one and submit it here.
Yours sincerely

### CODE REVIEW

**Comment by rmzelle** [comment]
Thanks.

Would it be acceptable to change the title to `<title>Presses Universitaires de Rennes - Archéologie et Culture (French)</title>`, and the file name to "presses-universitaires-de-rennes-archeologie-et-culture.csl"?

**Comment by FrancoisC37** [comment]
Yes, i do the changes
Thanks

File: presses-universitaires-de-rennes-archeologie-et-culture.csl [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+            <text variable="number-of-volumes" prefix="&#160;(" suffix="&#160;"/>
+            <text variable="volume" suffix=")" />
+          </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <text variable="collection-title" prefix="coll. " quotes="true"/>
+    <text variable="collection-number" prefix=", "/>
+  </macro>
```
Comment (rmzelle) [inline presses-universitaires-de-rennes-archeologie-et-culture.csl]: @FrancoisC37, there is another macro with the same "collection" name below, which is not allowed. Can you delete the one you don't need?

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**